lib/Dialogflow.ts

```diff
@@ -40,19 +39,11 @@ class DialogflowClass {
             };
 
             const queryParams = {
-                timeZone: 'America/Los_Angeles',
-                parameters:  {
+                timeZone: 'America/Los_Angeles', parameters:  {
```

Comment:
@AlexanderKanakis @ear-dev This is reverting changes of https://github.com/WideChat/Apps.Dialogflow/pull/97. Can you make sure it is necessary?

Reply:
Good catch @Shailesh351 ...........  @AlexanderKanakis can you verify please?  Thanks.